tests/slow/src/test/scala/tests/debug/MetalsDebugSuite.scala

```diff
@@ -0,0 +1,45 @@
+package tests.debug
+
+import java.net.URI
+
+object MetalsDebugSuite extends MetalsBaseDebugSuite {
+  testDebug("launch")(
+    layout = s"""/metals.json
+                |{ "a": {} }
+                |/a/src/main/scala/PrintHelloWorld.scala
+                |object PrintHelloWorld {
+                |  def main(args: Array[String]): Unit = {
+                |    println("Hello, World!")
+                |  }
+                |}""".stripMargin,
+    act = server => {
+      server.launch("a/src/main/scala/PrintHelloWorld.scala")
+    },
+    assert = client => {
+      assertEquals(client.output.stderr, "")
+      assertEquals(client.output.stdout, "Hello, World!")
+    }
+  )
+
+  testDebug("working-dir")(
```

Comment:
Can we add a test case for the situation when the project has a compile error? It should be clear to the user that running doesn't work because there's a compile error. Currently, an internal exception is thrown

<img width="447" alt="Screenshot 2019-05-20 at 17 19 35" src="https://user-images.githubusercontent.com/1408093/58032710-7b0c8280-7b23-11e9-8acd-6a5962f8e0eb.png">

Reply:
I think we could ask the user what to do via `window/showMessageRequest`

> This project can't be run because it has compile errors: Run anyways | Show errors | Cancel

Selecting `Run anyways` can use the old classpath and `Show errors` can trigger the `"metals-diagnostics-focus"` client command.